src/_gettsim/transfers/arbeitsl_geld_2/kost_unterk.py

```diff
@@ -29,8 +29,8 @@ def arbeitsl_geld_2_kost_unterk_m_bg_bis_2022(
 
 @policy_info(start_date="2023-01-01", name_in_dag="arbeitsl_geld_2_kost_unterk_m_bg")
 def arbeitsl_geld_2_kost_unterk_m_bg_ab_2023(
-    bruttokaltmiete_m_bg: float,
-    heizkosten_m_bg: float,
+    anteil_bruttokaltmiete_m_bg: float,
```

Comment:
```suggestion
    bruttokaltmiete_m_bg: float,
```
Or is there any issue with that? `anteil` sounds too much like a fraction compared to the amount. In the end, this is just a way to split up the `_hh` version; the other way round, aggregation should work as expected even. So just our usual aggregation logic, why give it a different name?

Reply:
We want to keep `bruttokaltmiete_m_hh' as the input variable. Defining a function `bruttokaltmiete_m' that splits up Bruttokaltmiete by the number of people in the household works perfectly fine. However, If the user forgets to add any of the two variables as an input, there will be an error message that reports a cycle in the DAG rather than the missing input. 

But I agree, a different word than `anteil’ would be better.

Reply:
I'd say we leave it at `bruttokaltmiete_m` then alongside opening an issue to catch such cycles.